estuary/ui
# Invariable Store Removal : Side Panel Store

## Issues

https://github.com/estuary/ui/issues/1088

## Changes

### 1088

- Moved the 'Side Panel` store into hook
- Cleaned up old hooks (was able to fully remove hooks file)

## Tests

### Manually tested

- Open and closed side panel
- See state stayed closed or open
- Switched connectors to see panel udpated

### Automated tests

- n/a

## Screenshots
